docs/delegation_design_spec/delegation_design_spec.tex

```diff
@@ -2224,7 +2219,67 @@ \subsection{Stakepool Registration}
 the present version, however, these pools will have to be created
 manually. This seems justified, given that there is only a handful of
 such very large stakeholders and seeing as such a feature would
-unnecessarily complicate engineering.
+complicate engineering.
+
+\subsection{Stakepool Metadata}
+\label{stakepool-metadata}
+
+The stake pool registration certificate can contain a URL (up to 32 bytes),
+pointing to a JSON object with the following content:
+
+\begin{description}
+\item[A Ticker]
+  of 3-5 characters, for a compact display of stake pools in a
+  wallet.
+\item[A Title/Name]
+  of up to 50 characters.
+\item[A Short Textual Description]
+\item[A URL]
+  to a homepage with additional information about the pool.
+\item[A list of IP addresses and/or Domain Names]
+  of public relay nodes that the stake pool operator provides to contribute to
+  the Cardano network.
+\item[A set of tags]
+\end{description}
+All characters in the metadata will be UTF8 encoded. The metadata is restricted
+to have a size of no more than 512 bytes.
+
+The stake pool operators are responsible for serving this data at the URL
+provided in the stake pool registration certificate. However, wallets will not
+retrieve the data for each stake pool at those individual URLs. Not only would
+that be inefficient, it would also allow malicious actors to intentionally slow
+down all wallets by intentionally delaying the response of their server.
+Instead, metadata will be cached on \emph{metadata proxy servers}.
+
+Those proxy servers will query the metadata URLs in the stakepool registration
+certificates, and cache the metadata, with the pool's \(sks\) as key.
+Invalidating the cache will be triggered when the content hash listed in the
+certificate does not match the content hash of the cached metadata. The wallet
+will then retrieve the metadata for pools it needs to display from one of the
+proxy servers, instead of having to send a request to each of the pool's
+metadata URLs.
+
+Those servers are simple, and in particular, they require relatively little
+trust: because of the content hash, someone running a proxy server can not
+display forged metadata.\todo{Is this actually sufficient, or should
+  we require the metadata to be signed by the pool's key?} The worst thing they
```

Comment:
The hash of the metadata is witnessed by the pool's operator key. I'm pretty sure that's sufficient.

Reply:
Ok, I'll remove the TODO. Thanks!